### PR TITLE
Apply up to PHP 7.4 syntax

### PR DESCRIPTION
**Pull request by samsonasik**
Signed-off-by: Abdul Malik Ikhsan <samsonasik@gmail.com>

|    Q          |   A
|-------------- | ------
| QA            | yes

### Description

Since this components require php 7.4, we can apply up to php 7.4 syntax.

### CODE REVIEW

File: test/TestAsset/commontypes.php [hunk 1]
```diff
@@ -612,10 +612,7 @@ class PublicPrivateProtected
      */
     protected $bar;
 
-    /**
-     * @var string
-     */
-    private $baz;
+    private string $baz;
```
Comment (Ocramius) [inline test/TestAsset/commontypes.php]: Careful: this is a test asset, unsure if this has an effect on test behaviour.

Comment (samsonasik) [reply test/TestAsset/commontypes.php]: ok, reverted 👍

**Comment by Ocramius** [comment]
Thanks @samsonasik! 

I'll hold off a minor release here though, as it's only test suite changes.

**Comment by samsonasik** [comment]
@Ocramius the `src` directory seems already applied the syntax update.

**Comment by Ocramius** [comment]
Yes, I mean that this diff is mostly about infrastructural/maintenance changes, so a release is only due after something else also lands.

**Comment by froschdesign** [comment]
@samsonasik @Ocramius 
Reminder:

> This package is considered feature-complete, and is now in **security-only** maintenance mode, following a [decision by the Technical Steering Committee](https://github.com/laminas/technical-steering-committee/blob/2b55453e172a1b8c9c4c212be7cf7e7a58b9352c/meetings/minutes/2020-08-03-TSC-Minutes.md#vote-on-components-to-mark-as-security-only).
> If you have a security issue, please [follow our security reporting guidelines](https://getlaminas.org/security/).
> If you wish to take on the role of maintainer, please [nominate yourself](https://github.com/laminas/technical-steering-committee/issues/new?assignees=&labels=Nomination&template=Maintainer_Nomination.md&title=%5BNOMINATION%5D%5BMAINTAINER%5D%3A+%7Bname+of+person+being+nominated%7D)
>
> If you need an actively developed SOAP client, we recommend [phpro/soap-client](https://github.com/phpro/soap-client).

https://github.com/laminas/laminas-soap#laminas-soap

---
There are other packages and [topics](https://github.com/orgs/laminas/projects/16) that need a little attention. 😉

**Pull request merged**